build.gradle.kts

```diff
@@ -4,11 +4,34 @@ plugins {
 }
 
 configurations {
+  implementation { exclude(module = "spring-boot-starter-web") }
```

Comment:
Oh .. are these here because of clashes?

Reply:
Confirmed - specifically this is to prevent non-reactive code form been executed